pkg/providers/instancetype/suite_test.go

```diff
@@ -813,6 +813,60 @@ var _ = Describe("InstanceType Provider", func() {
 			}
 		})
 
+		It("should support individual instance type labels", func() {
+			ExpectApplied(ctx, env.Client, nodePool, nodeClass)
+
+			nodeSelector := map[string]string{
+				// Well known
+				v1.LabelTopologyRegion:           "test-region-1",
+				corev1beta1.NodePoolLabelKey:     nodePool.Name,
+				v1.LabelTopologyZone:             "test-zone-1a",
+				v1.LabelInstanceTypeStable:       "Standard_NC24ads_A100_v4",
+				v1.LabelOSStable:                 "linux",
+				v1.LabelArchStable:               "amd64",
+				corev1beta1.CapacityTypeLabelKey: "on-demand",
+				// Well Known to AKS
+				v1alpha2.LabelSKUName:                      "Standard_NC24ads_A100_v4",
+				v1alpha2.LabelSKUFamily:                    "N",
+				v1alpha2.LabelSKUVersion:                   "4",
+				v1alpha2.LabelSKUConfidential:              "false",
```

Comment:
@tallaxes  https://github.com/search?type=code&q=LabelSKUConfidential we don't add this label  or isolatedSize to the instancetype requirements so these tests won't work with those labels. Thoughts on removing them? I don't see them used anywhere

Reply:
Corrent, neither selectors are supported yet (and they are not in the public docs), should not be in WellKnowLabels. Cleanup was on the TODO list, but great to have a test that catches this! 👍

Reply:
Removed them in this PR.